### DISCUSSION TITLE
liquibase waiting for changelog lock

### DISCUSSION THREAD

**Discussion by DongJunHan** [naver/ngrinder#969] [Q&A]
안녕하세요 "waiting for changelog lock.." lock을 해제하는 방법을 알고 싶습니다.

테스트 환경
os: Macbook Air M1
ngrinder docker version: 3.5.5-p1
jdk version: jdk 1.8.0_201

도커(docker-compose)를 사용하여 하나의 controller와 두 개의 agent를 구성하여 부하테스트를 진행중인데요
퍼포먼스 테스트를 진행하다보면 가끔 Ready상태에서 다음 단계로 오랜시간 기다려도 넘어가지 가지 않아 Agent컨테이너를 중지시키고 테스트를 삭제 후, 테스트를 재시도 해보는데요.
그것도 안되면 Controller도 중지했다가 재가동하여 시도해보려고 하는데,  테스트중간에 비정상적으로 꺼서 그런지 DB lock이 걸려서 controller컨테이너가 올라가지 않더라구요.
이전 퍼포먼스 결과 값을 유지한채로 DB lock을 해제하고 싶은데요.
DB lock을 해제하는 방법을 찾아보니 DB에 접속하여 테이블 컬럼 값 변경하라는데 DB접속 방법이 있을까요? 아니면 다른 방법이 있을지 조언 부탁드립니다. 

_컨테이너를 생성시 volume을 삭제하고 다시 만들면 정상적으로 올라옵니다 하지만 db가 없어져서 이전 퍼포먼스 결과 데이터가 전부 날라가서  pertest 대시보드에 뜨지 않습니다._

**Comment by moto6** [comment]
저도 동일한 증상을 겪고 있습니다. 3.5.5-p1, 3.5.8 버전에서 발생하며 docker-compose & docker 커맨드에서 발생합니다.  지금 사용중인 유일한 해결책은 docker 환경이 아닌 os 베어메탈 서버(vm 환경) 에서 war 파일을 실행하는것 뿐 입니다.
